src/app/forms/templates/test-records/section-templates/vehicle/group-3-light-vehicle-section.template.ts

```diff
@@ -0,0 +1,76 @@
+import { ValidatorNames } from '@forms/models/validators.enum';
+import { FormNode, FormNodeEditTypes, FormNodeTypes, FormNodeViewTypes, FormNodeWidth } from '@forms/services/dynamic-form.types';
+import { getOptionsFromEnum } from '@forms/utils/enum-map';
+import { ReferenceDataResourceType } from '@models/reference-data.model';
+import { EuVehicleCategories } from '@models/vehicle-tech-record.model';
+
+export const VehicleSectionGroup3: FormNode = {
+  name: 'vehicleSection',
+  label: 'Vehicle Details',
+  type: FormNodeTypes.GROUP,
+  viewType: FormNodeViewTypes.SUBHEADING,
+  children: [
+    {
+      name: 'vin',
+      label: 'VIN/chassis number',
+      value: '',
+      disabled: true,
+      type: FormNodeTypes.CONTROL,
+      viewType: FormNodeViewTypes.HIDDEN,
+      editType: FormNodeEditTypes.HIDDEN
+    },
+    {
+      name: 'trailerId',
+      label: 'Trailer ID',
+      value: '',
+      disabled: true,
+      width: FormNodeWidth.L,
+      type: FormNodeTypes.CONTROL,
+      viewType: FormNodeViewTypes.HIDDEN,
+      editType: FormNodeEditTypes.HIDDEN
+    },
```

Comment:
Having the trailer id on here doesn't seem quite right

Reply:
Removed.